lametro/templatetags/lametro_extras.py

```diff
@@ -175,5 +176,28 @@ def get_highlighted_attachment_text(context, ocd_id):
     attachment_text = ' '.join(d.full_text for d in bill.documents.all() if d.full_text)
 
     highlight = ExactHighlighter(context['query'])
-    
+
     return highlight.highlight(attachment_text)
+
+
+def get_event_occured(action):
```

Comment:
this name is a little confusing to me. how about more descriptive, like `get_action_event`?